homeassistant/components/iotty/__init__.py

```diff
@@ -0,0 +1,55 @@
+"""The iotty integration."""
+from __future__ import annotations
+
+import asyncio
+import logging
+
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.const import Platform
+from homeassistant.core import HomeAssistant
+from homeassistant.helpers import aiohttp_client, config_entry_oauth2_flow
+
+from . import api
+from .const import DOMAIN
+
+_LOGGER = logging.getLogger(__name__)
+
+# Uncomment these when developing
+# _LOGGER.setLevel(logging.DEBUG)
+# logging.getLogger("iottycloud").setLevel(logging.DEBUG)
```

Comment:
```suggestion
```

Please remove any commented out code.
n this specific case, the logger can be set to debug via https://www.home-assistant.io/integrations/logger/

Reply:
Thanks! Will next commits of this PR